DESIGNREFERENCE.md

```diff
@@ -0,0 +1,12 @@
+# Project-Tactics use case
+As a player I download the entire repository, create my own battles with my own enemies and maps and *share* this new battles with other players. So other players can play new battles, new characters, new jobs and much more. 
+
+## Project-Tactics (Open Tactica) Design References
+
+The intent of this project is to replicate the gameplay battle of Final Fantasy Tactics. So here some references that we use to develop this project.
```

Comment:
replicate -> "take inspiration from".

So here "are" some references to look at while developing this project.